as_perimenoun_oi_gynaikes.ass

```diff
@@ -947,13 +948,13 @@ Dialogue: 0,1:09:18.41,1:09:19.14,Default,,0,0,0,,Kokoretsi.
 Dialogue: 0,1:09:19.14,1:09:20.24,Default,,0,0,0,,Kokoretsi?
 Dialogue: 0,1:09:20.24,1:09:21.49,Default,,0,0,0,,Leave it up to me.
 Dialogue: 0,1:09:21.49,1:09:24.61,Default,,0,0,0,,Panos, he does not want normal Kokoretsi, he wants it Papadits-style.
-Dialogue: 0,1:09:24.61,1:09:27.34,Default,,0,0,0,,Papaditsa-style, of course. Leave it up to me I said.
+Dialogue: 0,1:09:24.61,1:09:27.34,Default,,0,0,0,,Papaditsa-style, of course. Leave it up to me I said!
 Dialogue: 0,1:09:37.06,1:09:41.41,Default,,0,0,0,,{\i1}I will become the step that will help you rise.{\i0}
 Dialogue: 0,1:09:41.41,1:09:45.79,Default,,0,0,0,,{\i1}I will become your cry of passion.{\i0}
-Dialogue: 0,1:09:45.79,1:09:50.91,Default,,0,0,0,,{\i1}You'll love me you'll see, you'll love me.{\i0}
+Dialogue: 0,1:09:45.79,1:09:50.91,Default,,0,0,0,,{\i1}You'll love me, you'll see, you'll love me.{\i0}
 Dialogue: 0,1:09:50.91,1:09:56.06,Default,,0,0,0,,{\i1}I will laugh for me. I will pay for me.{\i0}
 Dialogue: 0,1:09:56.06,1:10:00.94,Default,,0,0,0,,{\i1}Because, until this day...{\i0}
-Dialogue: 0,1:10:00.94,1:10:02.94,Default,,0,0,0,,Two thousand one hundred seventy.
+Dialogue: 0,1:10:00.94,1:10:02.94,Default,,0,0,0,,{\i1}...everything I did, I did for you!{\i0}\N2,170.

```

Comment:
I deem the dialogue more important here than the song. We go for comedic effect.

Reply:
I find the punchline of the song more funny that the money counting.
money counting joke is being built up here and manifests in the following lines...
That's why I left the number for readability but we can keep the punchline along with "Two thousand one hundred seventy"

Reply:
OK, let's add the song line then.